### ISSUE TITLE
Filename is not on the defineSubtitlesHandler extra property

### ISSUE THREAD

**Issue by daniel-lxs** [Stremio/stremio-addon-sdk#248]
According to this: https://github.com/Stremio/stremio-addon-sdk/blob/master/docs/api/requests/defineSubtitlesHandler.md#extra-parameters

The argument "extra" passed to defineSubtitlesHandler should get a "filename" property however I'm getting undefined when trying to access it:
![image](https://github.com/Stremio/stremio-addon-sdk/assets/57051444/51effc72-4dbd-40cc-b9ea-9c0ce1de873d)

Is there any way around this? Without this property I can't search subtitles based on keywords in the filename thus having to try each subtitle one by one instead of just getting the one for that specific file.

Thank you.

**Comment by jaruba** [comment]
On what platform are you testing? We are still working on getting the same behavior across all apps for subtitle addons, we are also including new behavior hints for streams related to this: https://github.com/Stremio/stremio-addon-sdk/pull/249

**Comment by daniel-lxs** [comment]
Thank you for answering so fast. I'm on Linux and tried the desktop and web version, neither send the filename to the plugin's endpoint.

**Comment by dexter21767-dev** [comment]
afaik this is only returned for torrent streams and not used for HTTP streams. are u testing using an HTTP stream (debrid for example)?

**Comment by daniel-lxs** [comment]
Yes I am using a debrid service, is it not possible to get the filename from a HTTP stream?
